backend/src/core/models/user/user.ts

```diff
@@ -32,6 +32,7 @@ export class User extends Model<User> {
       isEmail: true,
       isLowercase: true,
     },
+    unique: true,
```

Comment:
We might not need this if we use `id` instead

Reply:
isn't it good to have this additional constraint anyway, in case some part of the backend code is screwed up or someone manually modifies the db by mistake

Reply:
haha yea still good to have